web/client/static/fontawesome6/css/regular.css

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com
```

Comment:
General question: is it typical to have so much `fontawesome` checked directly into our own repo?  I'm surprised this isn't made available as a library package of some kind.

Reply:
We looked into this, and the current deployment is what `fontawesome` recommends.

This has the additional potential performance benefit that the `fontawesome` library may already be cached by the browser.